<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainApplicationUI</class>
 <widget class="QWidget" name="MainApplicationUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/background2.jpg);</string>
  </property>
  <widget class="QWidget" name="horizontalWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 0, 0);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 75 22pt &quot;微软雅黑 Light&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ffffff;&quot;&gt;物联网创新设计&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="TempBackView">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>51</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#TempBackView{
	border-style:outset;
	border-width:1;
	border-color: rgb(255, 255, 255);
	
	background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="temp_label">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgba(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;温度&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="temp_count">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>加载中</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="humidityBackView">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>51</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#humidityBackView{
	border-style:outset;
	border-width:1;
	border-color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="humi_label">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;湿度&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="humi_count">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>加载中</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="LightBackView">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>51</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#LightBackView{
	border-style:outset;
	border-width:1;
	border-color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="light_label">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;光照&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="light_count">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>加载中</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="verticalFrame">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>209</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="card_id">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>卡号</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>249</y>
     <width>359</width>
     <height>49</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame{
	border-style:outset;
	border-width:1;
	border-color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QSlider" name="steer_test">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>26</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="steer_count_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>7</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>54</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;90&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ledbutton">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>297</y>
     <width>180</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
color: rgb(255, 255, 255);
border-width:1;
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>LED</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bpmbutton">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>297</y>
     <width>180</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
border-width:1;
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>蜂鸣器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="redlightbutton">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>341</y>
     <width>180</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
border-width:1;
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>设置红外联动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="soundbutton">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>341</y>
     <width>180</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
border-width:1;
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>设置声音联动</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>382</y>
     <width>180</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="steer_set">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>-3</y>
      <width>180</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset;
border-width:1;
border-color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>382</y>
     <width>180</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="cardid_set">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-3</y>
      <width>180</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset;
border-width:1;
border-color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="setbutton1">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>414</y>
     <width>361</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
border-width:1;
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>点击设置舵机和卡值</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="temp_max">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>449</y>
     <width>181</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
border-width:1;
border-color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="temp_min">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>449</y>
     <width>181</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
border-width:1;
border-color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="humi_max">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>484</y>
     <width>181</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
border-width:1;
border-color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="light_max">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>519</y>
     <width>181</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
border-width:1;
border-color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="humi_min">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>484</y>
     <width>181</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
border-width:1;
border-color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="light_min">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>519</y>
     <width>181</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
border-width:1;
border-color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="setbutton2">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>554</y>
     <width>361</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:outset;
border-width:1;
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>设置传感器上下限</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>589</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255,0);
border-style:outset;
border-width:1;
border-color: rgb(255, 255, 255,0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="out1button">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>130</y>
     <width>120</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);

border-style:outset;
border-width:1;
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>


加载中</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>140</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>输出一</string>
   </property>
  </widget>
  <widget class="QPushButton" name="out2button">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>130</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);

border-style:outset;
border-width:1;
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>


加载中</string>
   </property>
  </widget>
  <widget class="QPushButton" name="out3button">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>130</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-style:outset;
border-width:1;
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>


加载中</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>输出二</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>输出三</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
